spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :user do
+    name  'Bob'
+    email	'some@mail.com'
```

Comment:
This can cause an issue in case you create two users in the same tests since you have a validation of uniqueness

Reply:
please use sequences